final_task/README.md

```diff
@@ -1,3 +1,31 @@
-# Your readme here
-Some text.
-Checkout how to write this file using *markdown*.
+# RSS-READER
+
+### Command-line utility which receives RSS URL and prints results in human-readable format.
+
+**Example:**
+python rss_reader.py https://news.yahoo.com/rss - -limit 1
+
+**Output**:
+
+Feed:  Yahoo News - Latest News & Headlines
+
+Title:  Thanks to Trump, the U.S. hasn&#39;t admitted a single refugee since September
+
+Date:  Fri, 01 Nov 2019 20:10:06 -0400
+
+Link:  https://news.yahoo.com/thanks-to-trump-the-us-hasnt-admitted-a-single-refugee-since-september-001006863.html
+
+Description:  October was the first full month in at least 18 years in which the United States did not admit any refugees.
+
+```
+positional arguments:
+  source         RSS URL
+
+optional arguments:
+  -h, --help     show this help message and exit
+  --version      Print version info
+  --json         Print result as JSON in stdout
+  --verbose      Outputs verbose status messages
+  --limit LIMIT  Limit news topics if this parameter provided
+
+```
```

Comment:
It would be nice to get here some details of your implementation or your thoughts about certain things in your code.

Reply:
Fixed

Reply:
There is still no information about how do tou store cached news